cluster/gce/manifests/cloud-controller-manager.manifest

```diff
@@ -23,7 +23,7 @@
 "containers":[
     {
     "name": "cloud-controller-manager",
-    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v1.25.2-alpha_ae91c1fc0c443c464a4c878ffa2a4544483c6d1f",
+    "image": "gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:v26.2.4",
```

Comment:
Unfortunately I still had to push this tag manually as we don't publish this image yet as part of the cloud-provider-gcp release process. cc @jprzychodzen

Reply:
Can you add commands how you push image to https://github.com/kubernetes/cloud-provider-gcp/blob/master/docs/instruction-how-to-bump-repository.md ?